config/7.10.0/common.yaml

```diff
@@ -120,12 +120,12 @@ console:
                     value: "[[.KeyStorePassword]]"
                   - name: WORKBENCH_ROUTE_NAME
                     value: "[[.ApplicationName]]-[[.Console.Name]]"
+                  - name: KUBERNETES_NAMESPACE
+                    value: "${KUBERNETES_NAMESPACE}"
+                  - name: KUBERNETES_LABELS
+                    value: "${KUBERNETES_LABELS}"
                   - name: JGROUPS_PING_PROTOCOL
-                    value: "openshift.DNS_PING"
-                  - name: OPENSHIFT_DNS_PING_SERVICE_NAME
-                    value: "[[.ApplicationName]]-[[.Console.Name]]-ping"
-                  - name: OPENSHIFT_DNS_PING_SERVICE_PORT
-                    value: "8888"
+                    value: "kubernetes.KUBE_PING"
```

Comment:
please move all config change to the `config/7.11.0` configs instead. e.g. `config/7.11.0/common.yaml`

Reply:
done